packages/host/app/components/ai-assistant/message/index.gts

```diff
@@ -215,6 +218,14 @@ export default class AiAssistantMessage extends Component<Signature> {
             </div>
           {{/if}}
 
+          {{#if @resources.files.length}}
+            <div class='cards' data-test-message-files>
```

Comment:
Class name cards implies there will be cards here, but actually there are files. Would be good to rename the class.